package.json

```diff
@@ -1,39 +1,81 @@
 {
   "name": "flux-angular",
-  "version": "3.1.6",
+  "version": "4.0.0",
```

Comment:
While the behavior of flux-angular shouldn't change because of these changes, it is still a breaking change because if some oneexplicitly referenced `release/flux-angular.js` that will no longer work.